### PR TITLE
dev-cpp/ms-gsl: Version bump to 2.0.0

### PR DESCRIPTION
**Pull request by janhenke**
Closes: https://bugs.gentoo.org/666410
Package-Manager: Portage-2.3.49, Repoman-2.3.10

### CODE REVIEW

File: dev-cpp/ms-gsl/ms-gsl-2.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils vcs-snapshot
```
Comment (a17r) [inline dev-cpp/ms-gsl/ms-gsl-2.0.0.ebuild]: vcs-snapshot seems unnecessary, this is no snapshot and has a fairly predictable `S` dirname.

Comment (janhenke) [reply dev-cpp/ms-gsl/ms-gsl-2.0.0.ebuild]: Leftover from before upstream tagged any releases. Now removed.

**Comment by gyakovlev** [comment]
Hi,
something is wrong.


```sh
* Package:    dev-cpp/ms-gsl-2.0.0
 * Repository: gentoo
 * Maintainer: gentoo@taujhe.de proxy-maint@gentoo.org
 * USE:        abi_x86_64 amd64 elibc_glibc kernel_linux test userland_GNU
 * FEATURES:   compressdebug network-sandbox preserve-libs sandbox splitdebug test userpriv usersandbox
>>> Unpacking source...
>>> Unpacking ms-gsl-2.0.0.tar.gz to /tmp/portage/dev-cpp/ms-gsl-2.0.0/work
>>> Source unpacked in /tmp/portage/dev-cpp/ms-gsl-2.0.0/work
 * ERROR: dev-cpp/ms-gsl-2.0.0::gentoo failed (prepare phase):
 *   The source directory '/tmp/portage/dev-cpp/ms-gsl-2.0.0/work/ms-gsl-2.0.0' doesn't exist
 * 
 * Call stack:
 *            ebuild.sh, line  792:  Called __ebuild_main 'prepare'
 *   phase-functions.sh, line 1068:  Called __dyn_prepare
 *   phase-functions.sh, line  384:  Called die
 * The specific snippet of code:
 *              die "The source directory '${S}' doesn't exist"
 * 
 * If you need support, post the output of `emerge --info '=dev-cpp/ms-gsl-2.0.0::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=dev-cpp/ms-gsl-2.0.0::gentoo'`.
 * The complete build log is located at '/tmp/portage/dev-cpp/ms-gsl-2.0.0/temp/build.log'.
 * The ebuild environment file is located at '/tmp/portage/dev-cpp/ms-gsl-2.0.0/temp/environment'.
 * Working directory: '/tmp/portage/dev-cpp/ms-gsl-2.0.0/homedir'
 * S: '/tmp/portage/dev-cpp/ms-gsl-2.0.0/work/ms-gsl-2.0.0'
```

you need to adjust `${S}`, i have it unpacked into
`/tmp/portage/dev-cpp/ms-gsl-2.0.0/work/GSL-2.0.0`

**Comment by janhenke** [comment]
Yes, when I removed the vcs-snapshot eclass, I forgot it was still fixing the ${S}. Sorry about it.

**Comment by gyakovlev** [comment]
all good now, pushed. thanks!

**Pull request closed without merging**